mfr/extensions/tabular/render.py

```diff
@@ -119,7 +119,8 @@ def _populate_data(self, fp, ext):
                 self._renderer_tabular_metrics['importer'] = function.__name__
                 try:
                     return imported(fp)
-                except (KeyError, ValueError):
+                except (KeyError, ValueError) as err:
+                    logger.error('Encountered an unexpected error when trying to render a table: {}'.format(err))
```

Comment:
Fyi, my memory has failed me. I remember that `logger.error()` doesn't go to Sentry and that we need to use `sentry.log_message()`, which turns out to be my old memories about OSF. I might have givne you some half-wrong advice. 👍 for figure this out.